async_simple/Common.h

```diff
@@ -18,10 +18,18 @@
 
 #include <stdexcept>
 
+#ifdef _WIN32
+#define __builtin_expect(EXP, C) (EXP)
+#endif
+
 #define FL_LIKELY(x) __builtin_expect((x), 1)
 #define FL_UNLIKELY(x) __builtin_expect((x), 0)
```

Comment:
We could use `[[likely]]` one day.

Reply:
Actually msvc, gcc and clang12+ supported likely/unlikely, but ci clang version is clang11, will have a unknown-attributes warning. Maybe add a compile flag to avoid this warning errorr.